### PR TITLE
docs: add current state

### PR DESCRIPTION
**Pull request by fuzzylogic2000**

### CODE REVIEW

File: docs/currentState.md [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+
+## Idea Create
+In an idea challenge project page, a button to add an idea is shown at the end of the list when the user is allowed to do so. Clicking it brings the user to the IdeaCreate page.
+
+Already working:
+- posting ideas as json with
+- name
+- description as plain text
+- categories if they exist (but whenever the category is clicked we get `VirtualizedLists should never be nested inside plain ScrollViews with the same orientation - use another VirtualizedList-backed container instead.`)
+- labels if they exist
```
Comment (phillimorland) [inline docs/currentState.md:58]: labels are not yet styled correctly

File: docs/currentState.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+- using the camera to take a photo or adding photos from the local storage. Posting the image does not yet work, though.
+
+Not currently working/in progress:
+- error handling for posting an idea (we do not currently show the error the API gives us?!, but handle the errors in the IdeaCreate component)
+- posting formData. We need that to post images. PR starting that: https://github.com/liqd/zt-app/pull/134
+- having the descrition in it's own component with a bigger textfield to open and close
```
Comment (phillimorland) [inline docs/currentState.md:64]: however has minimum 20 char on it, needs updating?

**Review by phillimorland — COMMENTED** [review]
super cool!

**Comment by fuzzylogic2000** [comment]
@phillimorland I added your remarks, also the ones for @khamui s PR. And I changed the comment stuff as that will be fixed when the PR is merged. I think we can merge this now?

**Comment by khamui** [comment]
thanks @fuzzylogic2000 ! also thanks for reviewing the coding guide @phillimorland.
Good documentation for the next sprint app!

**Comment by Rineee** [comment]
Also thanks from me for documenting! Super cool! :100:

**Pull request merged**